.travis/linux/OpenBLAS/install.sh

```diff
@@ -1,33 +1,40 @@
 set -ex
 
+# remove exiting blas packages to prevent conflicts
+sudo apt-get remove libblas3gf libatlas3gf-base
+
 # fetch fortran to build OpenBLAS
 sudo apt-get update -qq
 sudo apt-get install -qq gfortran
```

Comment:
Yes, libblas3gf in Ubuntu 12.04 has a dependency on libblas-common, but the dependency tree is different than in Ubuntu Xenial, so it results in a circular dependency.